great_expectations/expectations/metrics/column_aggregate_metrics/column_distinct_values.py

```diff
@@ -19,7 +22,7 @@ class ColumnDistinctValues(ColumnAggregateMetricProvider):
     metric_name = "column.distinct_values"
 
     @column_aggregate_value(engine=PandasExecutionEngine)
-    def _pandas(cls, column, **kwargs):
+    def _pandas(cls, column, **kwargs) -> Set[Any]:
```

Comment:
Thanks for typing the output! Can you type the inputs?

Reply:
✅